### PR TITLE
Specify file location for jicofo.auth.URL

### PR DESCRIPTION
**Pull request by samueljohnson**
The file location for placing the config parameter org.jitsi.jicofo.auth.URL isn't clear in the original document. Other Jitsi Community users (e.g. ygramoel) also had this issue.

### CODE REVIEW

**Comment by Freddie-m** [comment]
You're creating a PR for an issue from 2yrs ago? Things have changed a lot. This is not an issue - should be closed.

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
@Freddie-m yes the config is different, but with that syntax, that's the correct file and it still works.

**Comment by samueljohnson** [comment]
@saghul Thanks for merging.

**Comment by Freddie-m** [comment]
@saghul can't say I agree. This was created off of a thread from almost 2yrs ago (https://community.jitsi.org/t/secure-domain-enabled-but-still-no-password-requested/30107/10). I don't remember if that configuration was never mentioned in the guide pre-updating (it had to have been mentioned somewhere), but config has now been moved to jicofo.conf and although adding this line in sip-communicator still works, having these duplicate configurations potentially creates issues. My understanding was that we were moving away from sip-communicator - in my opinion, the guide should only reflect this updated config (as in other cases where config parameters were moved). Nonetheless, I defer to you.

**Comment by saghul** [comment]
I have no idea where it originated from, and it's irrelevant.

Yes we are migrating away from sip-communicator properties files, but they still work and the documentation has not been updated  to the new syntax yet.

Arguably this change is good because it at least clarifies to which file the current docs apply.

This doesn't mean we want to eventually change the docs to the new syntax. One thing at a time.

**Comment by Freddie-m** [comment]
> the documentation has not been updated to the new syntax yet.

I believe it has:
Screen Shot 2021-11-06 at 2 17 28 PM

This is akin to the move from `config.json` to `jibri.conf`. Even though `config.json` continued to be supported for a while (probably still is), it was always made clear that a migration was happening and recommendation was to use `jibri.conf`. The updated guide now makes no mention whatsoever of `config.json` when discussing the config files for Jibri (although it appears a few instances of `config.json` were overlooked in other sections on the page - clearly an oversight).

Having these two together, to me, is perhaps even more confusing. It doesn't follow the basic tenets of technical guide writing. 

Screen Shot 2021-11-06 at 2 23 50 PM

We know that `jicofo.conf` now drives Jicofo, even if `sip-communicator.properties` still works. The preferred current and intended future state should always be highlighted in favor of the past state (even if the past state still works in the interim).

In conclusion, this isn't the way I would do it, but you have editorial privileges, I don't. So, I rest.

**Comment by saghul** [comment]
This isn't about privileges. You can make a PR too, in fact it would have probably taken less time than this discussion.
